iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergStorageHandler.java

```diff
@@ -695,25 +700,64 @@ public List<ColumnStatisticsObj> getColStatistics(org.apache.hadoop.hive.ql.meta
     ColumnStatistics emptyStats = new ColumnStatistics();
     if (snapshot != null) {
       return IcebergTableUtil.getColStatsPath(table, snapshot.snapshotId())
-        .map(statsPath -> readColStats(table, statsPath))
+        .map(statsPath -> readColStats(table, statsPath, null).get(0))
         .orElse(emptyStats).getStatsObj();
     }
     return emptyStats.getStatsObj();
   }
 
-  private ColumnStatistics readColStats(Table table, Path statsPath) {
+  @Override
+  public AggrStats getAggrColStatsFor(org.apache.hadoop.hive.ql.metadata.Table hmsTable, List<String> colNames,
```

Comment:
I think some word is missing from the method name. 
For... ?

Reply:
i've used existing API naming for aggr stats - `IMetaStoreClient#getAggrColStatsFor`